homework.js

```diff
@@ -0,0 +1,136 @@
+function summation(number) {
+    let sum = 1;
+    for(let i=1;i<=number;i++){
+        sum+=i
+    } 
+    console.log(sum);
+} //the sum of the numbers from one to the given number.
+function summationEven(number2) {
+    let sum = 0;
+    for(let i=1;i<=number2;i++){
+        if(i%2===0)
```

Comment:
Good work; Also there another solution: if we invert the boolean of ```0``` will be true.
```if(! (i % 2)){...}```